draft-ietf-httpbis-http2.xml

```diff
@@ -2601,11 +2601,12 @@ HTTP2-Settings    = token68
             received.
           </t>
           <t>
-            A <x:ref>SETTINGS</x:ref> frame can alter the initial flow control window size for all
-            streams in the "open" or "half closed (remote)" state. When the value of
-            <x:ref>SETTINGS_INITIAL_WINDOW_SIZE</x:ref> changes, a receiver MUST adjust the size of
-            all stream flow control windows that it maintains by the difference between the new
-            value and the old value.
+            In addition to changing the flow control window for future streams, a

```

Comment:
I know I provided the `future streams` terminology in my issue report, and I understand its use is helpful to distinguish that per stream windows are not required to be maintained for streams in the IDLE state.  However the `future streams` seems to be a new expression.  I'm wondering if it makes sense to re-word to be more consistent and avoid introducing new expressions?  What about something like:

`A SETTINGS frame can alter the initial flow control window size for streams with active flow control windows (i.e. "open", and "half closed (remote)) and when streams transition out of the "idle" state and active flow control windows are allocated.`